org.OpenT2T.Sample.SuperPopular.Helpers/js/index.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
```

Comment:
I was hoping the folder name could be only "helpers" and inside that folder, put a sub-folder called opent2t-translator-helper-wink (i.e. the name of the node module). Namespacing this folder name like the other translators is likely to confuse people about what this is.